src/Services/DataService/DataService.ts

```diff
@@ -377,7 +382,7 @@ export abstract class DataService<T extends IModelData, R = T> extends BaseServi
 
     epics.push(
       this.fetchAllEpic.bind(this),
-      this.fetchRecordEpic.bind(this),
+      fetchRecordEpic(this),
```

Comment:
Since it's just a function, it's just taking the "context" as an argument, which is the current service.  As far as I know, `bind` is required to maintain the value of `this` within the child function, so that the original `fetchRecordEpic` would take the parent's `this` by default, rather than one of its own.